lua/indent_blankline/init.lua

```diff
@@ -352,14 +379,25 @@ local refresh = function()
                     if offset + i == context_start then
                         context_indent = (indent or 0) + 1
                     end
+                    if not prev_indent or indent + (extra and 1 or 0) <= prev_indent + max_indent_increase then
+                        prev_indent = indent
```

Comment:
the other code here assumes `indent` could be undefined.
I actually think this can't happen anymore, but I am not 100% sure :sweat_smile: just to be safe, do `(indent or 0)`

Reply:
I'll just move it after the zero check.